critiquebrainz/frontend/templates/macros.html

```diff
@@ -220,3 +220,41 @@ <h4 style="margin-bottom:0;">{{ _('Rate this {}:'.format(entity_type_readable))
         -
     {% endif %}
 {% endmacro %}
+
+{% macro display_pagination(current_page, count, limit, page_info, parameter) %}
```

Comment:
You can change the parameter here to something like `**params` and invoke the macro normally like `{{ display_pagination(page, count, limit, 'review.browse', entity_type=entity_type) }}` which looks a bit cleaner.

Reply:
I tried doing this, but when we have multiple parameters it does not work. Like in case of artists page, we pass 2 parameters, `id` and `release_type`. So I found this thing as the most flexible way.

Reply:
Uh right, Sorry I had forgotten these are Jinja2 macros and not python functions. **kwargs do work with [Jinja2](https://stackoverflow.com/questions/13944751/args-kwargs-in-jinja2-macros) as use different magic. The current version is better than jinja's magic so this is fine in the current way. Thanks.